app/controllers/api/members_controller.rb

```diff
@@ -1,19 +1,20 @@
 class Api::MembersController < ApplicationController
 
   def create
-    begin
-      new_member = Member.create!({
-        email: params[:email],
-        name: params[:name],
-        country: params[:country],
-        postal: params[:postal]
-        })
-      new_member.send_to_ak
-      render json: { member: new_member.as_json }
-    rescue
-      # rescue errorerorrr
-      render json: { errors: "errors that get passed" }
+    member = Member.find_or_initialize_by(email: member_params[:email])
+    member.assign_attributes(member_params)
+    if member.save!
```

Comment:
`save!` will raise `ActiveRecord::RecordInvalid ` if the record is invalid. You want the non-bang version which returns true/false